Umbrella/umbrella-proxy/ingest/parser.yml

```
@@ -0,0 +1,60 @@
+name: umbrella-proxy
```

I suggest adding the `ignored_values` directive.
```suggestion
name: umbrella-proxy
ignored_values: ["-", " "]
```